dialog/form_test.go

```diff
@@ -90,7 +91,7 @@ func TestFormDialog_Hints(t *testing.T) {
 	test.NewApp()
 	defer test.NewApp()
 	test.ApplyTheme(t, internalTest.LightTheme(theme.DefaultTheme()))
-	w := test.NewWindow(nil)
+	w := software.NewWindow(nil)
 	w.SetFullScreen(true)
 
```

Comment:
this test is broken and I have no clue how or why. pretty much every other test is fine, and this one the failure is visually identical. So I've no clue what's going wrong here...